src/Stratis.Bitcoin.Features.Consensus.Tests/PeerBanningTest.cs

```diff
@@ -66,6 +66,30 @@ private async Task<(TestChainContext context, IPEndPoint peerEndPoint)> Initiali
             Assert.True(context.PeerBanning.IsBanned(peerEndPoint));
         }
 
+        [Fact]
+        public async Task NodeIsSynced_PeerSendsAMutatedBlockAndPeerDisconnected_AndAddressIsNull_ThePeerGetsBanned_Async()
+        {
+            await this.NodeIsSynced_PeerSendsABadBlockAndPeerDisconnectedAndAddressIsNull_ThePeerGetsBanned_Async(
```

Comment:
This looks weird now but I'm guessing you may have more tests in future using the same method?

Reply:
Yeah it does, just copied the way another test was done in the file even though not varying the way the blocks were bad for this one, it was easier